roles/react_client/templates/config.json.j2

```diff
@@ -30,5 +30,16 @@
       "url": "/version.txt",
       "header": "Last-Modified",
       "intervalSec": 300
+    },
+
+    "donations": {
+      "statusAccount": "{{ donations_status_account }}",
```

Comment:
we need some dummy values for these variables in `roles/react_client/default/main.yml` otherwise rendering will fail on e.g. candy

Reply:
another option is to wrap this block in if/endif block, similar to how beta_channel works

Reply:
Added a |default("") filter, will it work?